PMMain/Midas/MidasGUI/MouseCtrl.cpp

```diff
@@ -31,14 +31,73 @@ void MouseCtrl::setScrollRate(int rate)
 
 void MouseCtrl::setMinMoveXTimeDelta(unsigned int rate)
```

Comment:
To note, this work was in progress at this time, and is clearly incomplete. I was pursuing lots of ideas to make controlling the mouse more 'smooth', but none worked yet, so they are all commented. This work is to be continued.